src/geo/map.js

```diff
@@ -47,12 +45,16 @@ var Map = Model.extend({
     this.layers.bind('change:attribution', this._updateAttributions, this);
 
     if (this._dataviewsCollection) {
-
       // When new dataviews are defined, a new instance of the map needs to be created
       this.listenTo(this._dataviewsCollection, 'add', _.debounce(this._onDataviewAdded.bind(this), 10));
     }
   },
 
+  instantiateMap: function () {
+    this._initBinds();
```

Comment:
The _initBinds seems to a bunch of binds, not only for dataviews but also for layers, are we sure we want to postpone _all_ of them?